### ISSUE TITLE
WP-SCSS not auto-compiling even with WP_SCSS_ALWAYS_RECOMPILE set 

### ISSUE THREAD

**Issue by ghost** [ConnectThink/WP-SCSS#104]
In my TOP project direcotry
```
	cd /home/web/site1/
```

& my WP root dir is
```
	/home/web/site1/wp/public/blog
```

I created dirs for source scss, and generated css
```
	tree /home/web/site1/assets/
		assets/
		├── css
		└── scss
		    └── main.scss
```

where,
```
	cat scss/main.scss
		.test { color: red; }
```

In my wp-config I set
```
	define('WPSCSS_THEME_DIR', ABSPATH . '../../../assets/' );
	define( 'WP_SCSS_ALWAYS_RECOMPILE', true );
```

At WP-SCSS' Settings,
```
	Configure Paths
	Add the paths to your directories below. Paths should start with the root of your theme. example: "/library/scss/"
```

so I set, relative to WPSCSS_THEME_DIR,
```
	Scss Location: /scss/
	CSS Location:  /css/
```

& and Saved settings (no error).

After restarting webserver, I don't see any generated *.css, and I see no log in the /scss/ dir, or anywhere else that I can tell.

Is there some reason that that^ doesn't auto-compile?

I understand that it will NOT generate if there's an error, but how do I at least get a log to see?

**Comment by shadoath** [comment]
This points out a bug that was built into the implementation of the `wp_enque_scripts` of the plugin. I recently updated the Plugin to have the option to set alternate base directories for the SCSS/CSS files to load from. Now the options are: current theme, plugin, and uploads. While doing this I found[ an issue ](https://github.com/ConnectThink/WP-SCSS/pull/165/files#diff-2ccc11b1d0241444bc0e36b7b314b8d15834eac41b73c9511a5a1e06998b9af3L218)where enqueuing stylesheets was using the theme base folder as a hardcoded value. I'm betting this is exactly what was your issue as well. 
I would like to expand the new `base folder` option to have 'custom' which would allow an option to write your own path to be used as the base location. Similar to how you were defining the `WPSCSS_THEME_DIR`, but this will need to be an absolute path.
